osu.Game/Overlays/Settings/Sections/Input/KeyBindingsSubsection.cs

```diff
@@ -49,7 +49,8 @@ private void load(RealmAccess realm)
                 Add(new KeyBindingRow(defaultGroup.Key, bindings.Where(b => b.ActionInt.Equals(intKey)).ToList())
                 {
                     AllowMainMouseButtons = Ruleset != null,
-                    Defaults = defaultGroup.Select(d => d.KeyCombination)
+                    Defaults = defaultGroup.Select(d => d.KeyCombination),
+                    RulesetBindings = bindings
```

Comment:
Are you sure this works when changing a key binding after the subsection was loaded, and *then* attempting to use the same key in another action?